src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1995,4 +2003,15 @@ public int getLastContactInHours()
     {
         return lastContactInHours;
     }
+
+    @Override
+    public boolean isCanBeAutoDeleted()
+    {
+        return canBeAutoDeleted;
+    }
```

Comment:
Sorry I was talking about the setCanBeAutoDeleted, that doesn't seem to have neither override nor javadoc